hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -490,7 +490,8 @@ public final class ScmConfigKeys {
   public static final String OZONE_SCM_EVENT_CONTAINER_REPORT_THREAD_POOL_SIZE =
       OZONE_SCM_EVENT_PREFIX + "ContainerReport.thread.pool.size";
   public static final int OZONE_SCM_EVENT_THREAD_POOL_SIZE_DEFAULT = 10;
-
+  public static final int OZONE_SCM_EVENT_REPORT_QUEUE_WAIT_DEFAULT = 60000;
+  public static final int OZONE_SCM_EVENT_REPORT_EXEC_WAIT_DEFAULT = 120000;
```

Comment:
Please add some comment about this time unit.

Reply:
Add comment, its in milli second

Reply:
Suggest to add threshold in the name, that is 

OZONE_SCM_EVENT_REPORT_QUEUE_WAIT_DEFAULT -> OZONE_SCM_EVENT_REPORT_QUEUE_WAIT_THRESHOLD_DEFAULT

OZONE_SCM_EVENT_REPORT_EXEC_WAIT_DEFAULT -> OZONE_SCM_EVENT_REPORT_EXEC_WAIT_THRESHOLD_DEFAULT